### PR TITLE
Fixing wrong word spelling in api/client/network.go

### PR DESCRIPTION
**Pull request by haoshuwei**
There's a word spelling mistake in  api/client/network.go ,i fixed it .

Signed-off-by: Shuwei Hao haoshuwei24@gmail.com

### CODE REVIEW

**Comment by thaJeztah** [comment]
LGTM, thanks!

**Comment by runcom** [comment]
LGTM

**Comment by vdemeester** [comment]
LGTM :cat:

File: api/client/network.go [hunk 1]
```diff
@@ -237,7 +237,7 @@ func (cli *DockerCli) CmdNetworkInspect(args ...string) error {
 	return nil
 }
 
-// Consolidates the ipam configuration as a group from differnt related configurations
+// Consolidates the ipam configuration as a group from diffrent related configurations
```
Comment (vdemeester) [inline api/client/network.go]: s/diffrent/different/ isn't it ?

Comment (thaJeztah) [inline api/client/network.go]: oh! must've been :zzz: 

that's embarrassing that I didn't spot it

@haoshuwei can you fix that?

Comment (runcom) [inline api/client/network.go]: How bad is my English then...

Comment (haoshuwei) [inline api/client/network.go]: My bad English too... I have fixed it

**Comment by vdemeester** [comment]
Thanks @haoshuwei :wink: 
LGTM this time :stuck_out_tongue_closed_eyes:

**Comment by thaJeztah** [comment]
yay ~~LGMT~~ LGTM

**Pull request merged**